### PR TITLE
Complex,test: fix the adaptive learning and test

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by hzhurauski — APPROVED** [review]

File: api/Application.Infrastructure/CPManagement/CPManagementService.cs [hunk 1]
```diff
@@ -108,10 +104,6 @@ public PagedList<CourseProjectData> GetProjects(int userId, GetPagedListParams p
             {
                 var courseProjects = from cp in query
                                      let acp = cp.AssignedCourseProjects.FirstOrDefault()
-                                     where acp.Student.IsActive.HasValue &&
```
Comment (hawkrai) [inline api/Application.Infrastructure/CPManagement/CPManagementService.cs]: Why did we remove this?

Comment (ghost) [reply api/Application.Infrastructure/CPManagement/CPManagementService.cs]: it’s mistake

File: api/Application.Infrastructure/CPManagement/CPManagementService.cs [hunk 2]
```diff
@@ -84,13 +84,9 @@ public PagedList<CourseProjectData> GetProjects(int userId, GetPagedListParams p
...
                 var courseProjects = from cp in query
                                      let acp = cp.AssignedCourseProjects.FirstOrDefault()
-                                     where (acp.Student.LastName.Contains(searchString) ||
+                                     where acp.Student.LastName.Contains(searchString) ||
                                             cp.Theme.Contains(searchString) ||
-                                            acp.Student.Group.Name.Contains(searchString)) &&
-                                            acp.Student.IsActive.HasValue &&
-                                            acp.Student.IsActive.Value &&
-                                            acp.Student.Confirmed.HasValue &&
-                                            acp.Student.Confirmed.Value
```
Comment (hawkrai) [inline api/Application.Infrastructure/CPManagement/CPManagementService.cs:93]: I still don't understand why this should be removed?

**Review by hawkrai — CHANGES_REQUESTED** [review]

File: api/Application.Infrastructure/CPManagement/CPManagementService.cs (hunk 2 above)
Comment (ghost) [reply api/Application.Infrastructure/CPManagement/CPManagementService.cs:93]: this file does not need to be changed, I accidentally uploaded it

Comment (hawkrai) [reply api/Application.Infrastructure/CPManagement/CPManagementService.cs:93]: I still see the changes, roll them back

**Pull request merged**